README.md

```diff
@@ -202,7 +202,7 @@ Production capacities are centralized in the [zones.json](https://github.com/tmr
    - Biomass, Hydro, Solar, Wind: [IRENA](https://www.irena.org/-/media/Files/IRENA/Agency/Publication/2020/Mar/IRENA_RE_Capacity_Statistics_2020.pdf)
    - Nuclear, Gas: [wikipedia.org](https://en.wikipedia.org/wiki/List_of_power_stations_in_Armenia)
 - Aruba: [WEB Aruba](https://www.webaruba.com/energy-production/power-production-figures)
-- Australia [wikipedia.org](https://en.wikipedia.org/wiki/Wind_power_in_Australia#Wind_power_by_state)
+- Australia [OpenNEM](https://opennem.org.au/facilities/all/)
```

Comment:
Given that Western Australia isn't in the NEM, shouldn't you keep the Wikipedia link for that one?